### PR TITLE
Core Contracts deployed on  Moonbase Alpha Testnet

### PR DESCRIPTION
**Pull request by SuryaSan24x7**

### CODE REVIEW

**Review by nikakhachi — CHANGES_REQUESTED** [review]
fix conflicts please

File: docs/testnet-deployments.md [hunk 1]
```diff
@@ -551,6 +552,30 @@
 
 ## Okx Testnet
 
+| Contract           | Address                                    |
+| ------------------ | ------------------------------------------ |
+| BabelCore          | 0xed0ac2eea2cec5c27e509e29814574603587f682 |
```
Comment (nikakhachi) [inline docs/testnet-deployments.md]: The addresses got messed up. Check it please. The Okx testnet and  moonbase testnet addresses got messed up it seems.

**Review by nikakhachi — CHANGES_REQUESTED** [review]

File: docs/testnet-deployments.md (hunk 1 above)
Comment (SuryaSan24x7) [reply docs/testnet-deployments.md]: These are the correct addresses  just verified it on the both Chains.

**Review by SuryaSan24x7 — COMMENTED** [review]
Addresses Updated

**Pull request closed without merging**